kolibri/plugins/device/assets/src/views/SelectContentPage/index.vue

```diff
@@ -46,6 +47,13 @@
         >
           {{ $tr('problemTransferringContents') }}
         </UiAlert>
+        <UiAlert
+          v-show="transferFileSize > availableSpace"
+          :dismissible="false"
+          type="error"
+        >
+          {{ spaceTranslator.$tr('notEnoughSpaceForChannelsWarning') }}
+        </UiAlert>
```

Comment:
filed a follow-up issue regarding in-page errors: https://github.com/learningequality/kolibri/issues/8690